crates/matrix-sdk/src/sliding_sync/error.rs

```diff
@@ -19,8 +19,8 @@ pub enum Error {
     #[error("The sliding sync list `{0}` is handling a response, but its request generator has not been initialized")]
     RequestGeneratorHasNotBeenInitialized(String),
 
-    /// Someone has tried to modify a sliding sync list's ranges, but the
-    /// selected sync mode doesn't allow that.
+    /// Someone has tried to modify a sliding sync list's ranges, but only the
+    /// selected sync mode does allow that.
```

Comment:
This now contradicts the error message, no? Also why would it throw an error if the selected sync mode allows range modification?

Reply:
Looking at the room, I understand now.
```suggestion
    /// `Selective` sync mode allows this.
```